Detectors/TPC/workflow/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ o2_add_library(TPCWorkflow
                        src/TrackReaderSpec.cxx
                        src/RawToDigitsSpec.cxx
                        src/LinkZSToDigitsSpec.cxx
+                       src/TPCSectorCompletionPolicy.cxx
```

Comment:
I am not sure if it is ideal to have this in TPC/workflow. It pulls in quite some dependencies, while in fact only DataFormatsTPC and Framework/Core is needed. Could we perpaps put it somewhere else, or add a new library Detectors/TPC/workflow/helpers ?

Reply:
Agreed. however, the ITS matching wf depends already on TPCWorkflow. But I would take this as a task to be done. We need to optimize dependencies, i.e. reduce to the minimum.